pkg/provider-local/webhook/controlplane/ensurer.go

```diff
@@ -45,13 +44,6 @@ func (e *ensurer) EnsureKubeletConfiguration(_ context.Context, _ gcontext.Garde
 	return nil
 }
 
-func (e *ensurer) EnsureETCD(_ context.Context, _ gcontext.GardenContext, newObj, _ *druidv1alpha1.Etcd) error {
-	// Remove any affinities from etcd's spec because in the local setup we only have one node,
-	// so that pod (anti-) affinities can't apply here.
-	newObj.Spec.SchedulingConstraints.Affinity = nil
```

Comment:
How do we ensure that this does not break the regular e2e tests (with only a single-node kind setup)?

Reply:
Anyways in single-node kind cluster, we support only non-HA control plane. So there is no affinity rules.

Reply:
OK, so IIUC correctly, this was taken care of in `etcd-druid` (it does no longer set `SchedulingConstraints` for non-HA clusters). Thanks!